src/Tribe/Integrations/WPML/Linked_Posts.php

```diff
@@ -137,23 +170,171 @@ protected function get_curent_language_linked_posts_ids( array $args ) {
 		// the user might have posts that are *only* translated and none in the default language
 		$query = new WP_Query( $args );
 
-		return $query->have_posts() ? $query->posts : array();
+		$linked_post_ids = $query->have_posts() ? $query->posts : array();
+
+		$this->cache[ $cache_key ] = $linked_post_ids;
+
+		return $linked_post_ids;
 	}
 
 	/**
+	 * Returns a list of linked post IDs for the default language.
+	 *
+	 *
 	 * @param array $args An array WP_Query arguments
 	 *
 	 * @return array An array of linked posts filtered by the default language
 	 */
 	protected function get_default_language_linked_post_ids( array $args ) {
+		$cache_key = $this->cache->make_key( func_get_args(), 'default_language_linked_post_ids' );
```

Comment:
You cannot use `func_get_args()` directly into a function/method call, you need to save it in a variable first. PHP 5.2 compat.